### PR TITLE
FINERACT-1471: AsciiDoc extension for read/write/batch separation

### PR DESCRIPTION
**Pull request by taskain7**
## Description

The AsciiDoc has been extended for read/write/batch separation.

### CODE REVIEW

File: fineract-doc/src/docs/en/chapters/development/instance-type.adoc [hunk 1]
```diff
@@ -0,0 +1,178 @@
+= Fineract Instance types
+
+In cases where Fineract has to deal with high load, it can cause a performance problem for a single Fineract instance.
+To overcome this problem, Fineract instances can be started in different instance types for better scalability and performance in a multi-instance environment:
+
+Read instance
```
Comment (galovics) [inline fineract-doc/src/docs/en/chapters/development/instance-type.adoc]: Let's make this a proper list: https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/

File: fineract-doc/src/docs/en/chapters/development/index.adoc [hunk 2]
```diff
@@ -6,6 +6,8 @@ include::git.adoc[leveloffset=+1]
 
 include::gpg.adoc[leveloffset=+1]
 
+include::instance-type.adoc[leveloffset=+1]
```
Comment (galovics) [inline fineract-doc/src/docs/en/chapters/development/index.adoc]: I think this section should rather be in the architecture one (or maybe deployment). Thoughts?

**Review by galovics — CHANGES_REQUESTED** [review]
Other than those 2 comments, it's a good starting point. Let's fix 'em and we're good.

File: fineract-doc/src/docs/en/chapters/development/index.adoc (hunk 2 above)
Comment (taskain7) [reply fineract-doc/src/docs/en/chapters/development/index.adoc]: Yeah, it could be any of them, but I would vote for deployment rather than architecture.

**Review by galovics — APPROVED** [review]

**Pull request merged**